tools/nxstyle.c

```diff
@@ -1974,7 +1982,7 @@ int main(int argc, char **argv, char **envp)
 
                 case '/':
 
-                  /* C comment terminator */
+                  /* C comment terminator*/
```

Comment:
Ooops.. now I see a missing space before the comment terminator.  Ironic in this case.